releasenotes/notes/pip_proxy_env-8bf23f475b80a058.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Correctly pass the agent's proxy settings to pip when using TUF
```

Comment:
nit: could clarify the context here a bit? (mention the `datadog-agent integration` command for example). Because the message will be in the changelog without additional context :)